### ISSUE TITLE
Error in sublime v3124

### ISSUE THREAD

**Issue by robjhyndman** [LaTeXing/LaTeXing#226]
I've just updated to v3124 and now LaTeXing seems to be broken.

```
Error loading syntax file "Packages/LaTeXing/support/LaTeX Beamer.tmLanguage": Apparent recursion within a with_prototype action: 25000 context sanity limit hit
```

**Comment by umid** [comment]
I am also having problems after updating to v3124. 

Key menu items (Preferences -> Package Settings -> LaTeXing -> Documentation/Version/Buy License/Install License/Settings Extended) are all inactive. Ctrl+B and bibtex autocompletion also stopped working. The package was included in the list of "disabled_packages" after the update but removing it from there didn't help. Reinstalling the package didn't help either.

Any guidance on how to resolve this would be great. OS is Ubuntu 16.04 LTS.

**Comment by csch0** [comment]
Thanks for letting me know and providing so many details. Will try to verify and fix it asap.

**Comment by gapplef** [comment]
Sublime 3126 on Ubuntu Gnome 16.04
I have exactly the same problem with @umid, and also restart and reinstall won't help.
After seeing this problem, I open the console, try to find out what was happening.
I see some lines said missing dependency, and almost the same time, a popup window said "3 missing dependency was just installed. Sublime Text should be restarted". I restart the Sublime Text, and seems the Latexing is already active . But again a popup window said "1 missing dependency was just installed. Sublime Text should be restarted". Again, I restart the Sublime Text, and finally, Latexing work fine now!

So, in short, the problem is caused by missing dependency(but why new dependency?) , and nothing have to do, just open the console and  waiting for the Package manger to install the missing dependency.

**Comment by csch0** [comment]
Ok whats dependency is missing according to the console output?

**Comment by gapplef** [comment]
@csch0 Sorry, but I only remember the second time, missing dependency is "ssl-linux". For the first time, I think I saw "oauthlib", and don't remember others.
